src/reporter.ts

```diff
@@ -1,29 +0,0 @@
-import { version } from "../package.json";
-import type { TopsortEvent } from "./events";
-
-interface Config {
-  token: string;
-  url?: string;
-}
-
-export async function reportEvent(e: TopsortEvent, config: Config) {
-  try {
-    const url = (config.url || "https://api.topsort.com") + "/v2/events";
-    const r = await fetch(url, {
-      method: "POST",
-      headers: {
-        "Content-Type": "application/json",
-        // Can't use User-Agent header because of
-        // https://bugs.chromium.org/p/chromium/issues/detail?id=571722
-        "X-UA": `ts.js/${version}`,
```

Comment:
Note that we will be loosing the ability to know which version of analytics.js they are using. It'd be great if we could add a parameter that gets added to the user agent sent by topsort sdk.